src/HeaderBar/index.js

```diff
@@ -18,21 +18,24 @@ import { Notifications } from './Notifications.js'
 import '../locales'
 import i18n from '@dhis2/d2-i18n'
 
+const query = {
+    systemInfo: {
+        resource: 'system/info',
+    },
+    user: {
+        resource: 'me',
+    },
+    apps: {
+        resource: 'action::menu/getModules',
+    },
+    notifications: {
+        resource: 'me/dashboard',
+    },
+}
+
 export const HeaderBar = ({ appName, className }) => {
-    const { loading, error, data } = useDataQuery({
-        systemInfo: {
-            resource: 'system/info',
-        },
-        user: {
-            resource: 'me',
-        },
-        apps: {
-            resource: 'action::menu/getModules',
-        },
-        notifications: {
-            resource: 'me/dashboard',
-        },
-    })
+    const config = useConfig()
```

Comment:
It looks like this was added but isn't used?

Reply:
Thanks, fixed in 3d80f68.